zeppelin-interpreter/src/main/java/com/teragrep/zep_01/interpreter/thrift/InterpreterRPCException.java

```diff
@@ -16,15 +16,15 @@
  * limitations under the License.
  */
 /**
- * Autogenerated by Thrift Compiler (0.20.0)
+ * Autogenerated by Thrift Compiler (0.15.0)
  *
  * DO NOT EDIT UNLESS YOU ARE SURE THAT YOU KNOW WHAT YOU ARE DOING
  *  @generated
  */
 package com.teragrep.zep_01.interpreter.thrift;
 
 @SuppressWarnings({"cast", "rawtypes", "serial", "unchecked", "unused"})
-@javax.annotation.Generated(value = "Autogenerated by Thrift Compiler (0.20.0)", date = "2024-12-05")
+@javax.annotation.Generated(value = "Autogenerated by Thrift Compiler (0.15.0)", date = "2025-03-13")
```

Comment:
please use the same 0.20.0 version of the Thrift Compiler as previously was used